proto/viam/app/data/v1/data.proto

```diff
@@ -186,51 +186,44 @@ message DeleteBinaryDataByIDsResponse {
   uint64 deleted_count = 1;
 }
 
-// Annotation specifies the ways in which we can associate annotations with data, e.g. with a text tag or image bounding box
-message Annotation {
-  string tag = 1;
-}
-
-// AddAnnotationsToBinaryDataByFileIDsRequest requests adding all specified annotations to each of the files when file IDs are provided
-message AddAnnotationsToBinaryDataByFileIDsRequest {
+// AddTagsToBinaryDataByFileIDsRequest requests adding all specified tags to each of the files when file IDs are provided
+message AddTagsToBinaryDataByFileIDsRequest {
   repeated string file_ids = 1;
-  repeated Annotation annotations = 2;
+  repeated string tags = 2;
 }
 
-// AddAnnotationsToBinaryDataByFileIDsResponse returns IDs of added annotations which can be passed to RemoveAnnotationsFromBinaryDataByFileIDs
-message AddAnnotationsToBinaryDataByFileIDsResponse {
-  repeated string annotation_ids = 1;
+// AddTagsToBinaryDataByFileIDsResponse returns nothing
+message AddTagsToBinaryDataByFileIDsResponse {
 }
 
-// AddAnnotationsToBinaryDataByFilterRequest requests adding all specified annotations to each of the files when a filter is provided
-message AddAnnotationsToBinaryDataByFilterRequest {
+// AddTagsToBinaryDataByFilterRequest requests adding all specified tags to each of the files when a filter is provided
+message AddTagsToBinaryDataByFilterRequest {
   Filter filter = 1;
-  repeated Annotation annotations = 2;
+  repeated string tags = 2;
 }
 
-// AddAnnotationsToBinaryDataByFilterResponse returns IDs of added annotations which can be passed to RemoveAnnotationsFromBinaryDataByFileIDs
-message AddAnnotationsToBinaryDataByFilterResponse {
-  repeated string annotation_ids = 1;
+// AddTagsToBinaryDataByFilterResponse returns nothing
```

Comment:
supernit for consistency: think these Add/Delete responses don't need documentation, and can be on one line like [this](https://github.com/viamrobotics/api/blob/main/proto/viam/service/vision/v1/vision.proto#L161)